project4/project/ebs/static/ebs/css/add-blog.css

```diff
@@ -14,4 +14,30 @@
 }
 .form-margin{
```

Comment:
remove this class, you can set margin to form-group class to avoid using multiple classes